CHANGELOG.md

```diff
@@ -9,6 +9,13 @@ See [RELEASE](./RELEASE.md) for workflow instructions.
 * [#5432](https://github.com/spacemeshos/go-spacemesh/pull/5419) Fixed a possible race that is caused by a node
   processing the same bootstrapped active set twice.
 
+## Release v1.3.2
+
+### Improvements
+
+* [#5394](https://github.com/spacemeshos/go-spacemesh/pull/5394) Add rowid to tables with inefficient clustered indices.
+  This reduces database size and improves its performance.
+
```

Comment:
We already have a v1.3.2 release, are the changes for that version missing?

Reply:
It was a typo, sorry, fixed